cmd/calyptia/auth.go

```diff
@@ -6,100 +6,12 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"io/fs"
-	"os"
-	"path/filepath"
-
-	"github.com/zalando/go-keyring"
 )
 
 const serviceName = "cloud.calyptia.com"
+const backUpFolder = ".calyptia"
```

Comment:
Things like this need to be captured in the docs - we need to ensure complete removal for package install for example. Where exactly is this folder (under a home area for a user or in the calyptia area)?

Please if you're adding extra directories or files then make sure you raise an issue on calyptia/core-images so we track the file/directory properly for packages. It's best practice and quite often a requirement for more secure systems.

Reply:
What happens if we run this as a user with no home directory?
Can we make it configurable?